tsc/process/security.md

```diff
@@ -0,0 +1,100 @@
+**Security and OpenVDB**
+
+The OpenVDB project takes security very seriously.
```

Comment:
"The OpenVDB Technical Steering Committee (TSC) takes security very seriously" might be a better way to phrase this.